modules/images-s2i-nodejs-pulling-images.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/using_images/using-images-source-to-image.adoc
+
+[id="images-s2i-nodejs-pulling-images_{context}"]
+= Pulling images for Node.js
+
+These images come in two options:
+
+* RHEL 8
+* CentOS 7
+
+*RHEL 8 Based Images*
+
+The RHEL 8 images are available through the Red Hat Registry.
+
+.Procedure
+
+* To pull the RHEL 8 image, enter the following command for the version of Node.js you want:
++
+.Node.js `0.10`
+[source,terminal]
+----
+$ podman pull registry.redhat.io/openshift4/nodejs-010-rhel7
+----
++
+.Node.js `4`
+[source,terminal]
+----
+$ podman pull registry.redhat.io/rhscl/nodejs-4-rhel8
+----
+
+*CentOS 7 Based Image*
+
+This image is available on link:quay.io[Quay.io].
+
+.Procedure
+
+* To pull the CentOS 7 image, enter the following command:
++
+[source,terminal]
+----
+$ podman pull openshift/nodejs-010-centos7
+----
+
+To use these images, you can either access them directly from registry.redhat.io, or push them into your {product-title} image registry. Additionally, you can create an image stream that points to the image, either in your container image registry or at the external location. Your {product-title} resources can then reference the
```

Comment:
All `image stream` in this pr need replace to `imagestream`

Reply:
Thank you. We have new guidance (https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#api-object-formatting) that changes the way we're writing "imagestream" to "image stream" with general references. Is that something that is going to cause trouble for users? If so, I can reach out and see if we can adjust the guidance.